resources/api/resource.py

```diff
@@ -244,6 +244,8 @@ class ResourceSerializer(
     pricing_user_groups = serializers.SerializerMethodField()
     pricing_event_types = serializers.SerializerMethodField()
 
+    include_other_event_type_option = serializers.SerializerMethodField(read_only=True)
```

Comment:
Isn't `SerializerMethodField` always read only?

Reply:
It is, yes. I just wanted to make it clear